cvat/requirements/base.in

```diff
@@ -2,10 +2,10 @@
 
 attrs==21.4.0
 click==8.1.3
-Django==3.2.18
-django-appconf==1.0.4
+Django~=4.2.1
+django-appconf==1.0.5
```

Comment:
I think django-appconf, rcssmin and rjsmin can be removed from this file entirely, because we don't use them directly (all three are dependencies of django-compressor).